package.json

```diff
@@ -17,6 +17,11 @@
     "lerna": "3.16.5"
   },
   "dependencies": {
-    "concurrently": "5.0.0"
+    "@fortawesome/fontawesome-svg-core": "^1.2.27",
+    "@fortawesome/free-solid-svg-icons": "^5.12.1",
+    "@fortawesome/react-fontawesome": "^0.1.8",
+    "concurrently": "5.0.0",
+    "react-grid-system": "^6.1.0",
+    "recharts": "^2.0.0-beta.1"
```

Comment:
pretty heavy library dependency wise ^^ but it looks good! don't think we care too much about bundle size atm